README.md

```diff
@@ -50,6 +50,7 @@ $ STORAGE=cassandra java -jar jaeger-spark-dependencies.jar `date -u -d '1 day a
 The following variables are common to all storage layers:
 
     * `SPARK_MASTER`: Spark master to submit the job to; Defaults to `local[*]`
+    * `DATE`: Date in YYYY-mm-dd format.
```

Comment:
Just wondering whether a more descriptive env var name may be appropriate? DEPENDENCY_DATE?